source/features/clean-checks-list.css

```diff
@@ -28,6 +28,20 @@
 	opacity: 50%;
 }
 
+/* Untruncate long check descriptions: https://github.com/hashicorp/consul/commit/38d94282ca0a1ed66d21ebb5c966a97e450d0ae3 */
+.merge-status-item .css-truncate {
+	white-space: unset !important;
+}
+
+/* Top align check symbol */
+.merge-status-item .merge-status-icon {
+	align-self: flex-start !important;
+}
+
+.merge-status-item .merge-status-icon svg {
```

Comment:
Uh, why is nested CSS not working? What browser and browser version are you testing this on? New Refined GitHub features should use both :has() and nested CSS wherever necessary

- https://github.com/refined-github/refined-github/issues/6665#issuecomment-1715221550
- https://github.com/refined-github/refined-github/issues/5797#issuecomment-1231273904

Reply:
Running the extension on https://github.com/refined-github/refined-github/pull/7038/commits/255b3672666f129d7bf012359637cb61606bf7fd works on sandbox, but not this PR

I think it might be an issue with chrome and how it handles `width` and `height`

![image](https://github.com/refined-github/refined-github/assets/58778985/10beb90b-4fc0-45c8-a17b-6a2b35789f15)

`height: 20px` is crossed out in the dev tools but the svg is in fact using the 20px height and not the 16px attribute

Firefox does show the height as 20px in the dev tools

Reply:
Try using this css:

```css
/* Untruncate long check descriptions: https://github.com/hashicorp/consul/commit/38d94282ca0a1ed66d21ebb5c966a97e450d0ae3 */
.merge-status-item .css-truncate {
	white-space: unset !important;
}

/* Top align check symbol */
.merge-status-item .merge-status-icon {
	align-self: flex-start !important;

	svg {
		height: 20px;
	}
}
```

the nested css just doesn't exist when checking with dev tools, specifically on the PR checks at the bottom ↓

Reply:
Oh the reason is this bug https://bugs.chromium.org/p/chromium/issues/detail?id=1427259

In short: use `& svg` as the nested selector

Reply:
Yeah, I was just look at that post; fixed in chromium but not yet in chrome, working on a fix rn